docs/pages/faq.mdx

```diff
@@ -63,6 +63,18 @@ are not logged in to Teleport or may not have access to a browser.
 Make sure that all Teleport components are at least at version 10.3.0. Older versions
 don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.
 
+## `tsh` is very slow on Windows, what to do?
+
+If your host machine is joined to an Active Directory domain, you might find user lookups take a lot longer than you expect because of the number of Active Directory accounts that must be scanned. To fix this issue, you can use environment variables to set default account information for your Teleport user. If you are experiencing long lookup times on Windows, do the following:
```

Comment:
- this is a very long line, can we break it up to roughly match the length used in the rest of the page?
- I would end the sentence after "longer than you expect" and start a new sentence that explains why.

Reply:
Done